### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by aron-net**
## Blog app - creating a data model

:heavy_check_mark: [Project Requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/creating_data_model_project_v1_1.md)

Learning objectives: :man_technologist:  :

:heavy_check_mark: Install Ruby on Rails framework.
:heavy_check_mark: Use Ruby Gems as a software packages system.
:heavy_check_mark: Understand what ORM is.
:heavy_check_mark: Use database migration files to maintain database schema.
:heavy_check_mark: Install and check linters.
:heavy_check_mark: Add a descriptive README file.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.integer 'users_id', null: false
+    t.index ['users_id'], name: 'index_posts_on_users_id'
+  end
+
+  create_table 'users', force: :cascade do |t|
+    t.string 'name'
+    t.string 'photo'
+    t.text 'bio'
```
Comment (rustam-tolipov) [inline db/schema.rb]: - [ ] In requirement you also need to have `posts_counter` as well. Kindly add `posts_counter` to your database :).

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    t.index ['posts_id'], name: 'index_likes_on_posts_id'
+    t.index ['users_id'], name: 'index_likes_on_users_id'
+  end
+
+  create_table 'posts', force: :cascade do |t|
+    t.string 'title'
+    t.text 'text'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.integer 'users_id', null: false
```
Comment (rustam-tolipov) [inline db/schema.rb]: - [ ] Kindly add `:comments_counter :likes_counter` to your posts as well.

File: README.md [hunk 3]
```diff
@@ -1 +1,88 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (rustam-tolipov) [inline README.md:94]: [OPTIONAL] To make your project professional, kindly add MIT license to your project.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @aron-helu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (aron-net) [reply README.md:94]: @Rustamxon7 Hello! Can you please check readme file? Already have implemented it.

File: db/schema.rb (hunk 2 above)
Comment (aron-net) [reply db/schema.rb]: comments_counter and likes_counter added to the posts table.

File: db/schema.rb (hunk 1 above)
Comment (aron-net) [reply db/schema.rb]: posts_counter added to the user table.

**Review by FrederickMih — APPROVED** [review]
# Approve :heavy_check_mark: 

Hi @aron-helu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :green_circle: 

- Descriptive PR :+1: 
- No Linter error :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (rustam-tolipov) [reply README.md:94]: Hello, Could you show the license, please? 😊 In your readme file, it supposes to work correctly. However, If I try to open it, it shows me a broken link.

**Pull request merged**